### DISCUSSION TITLE
Scale prompt window?

### DISCUSSION THREAD

**Discussion by PeterJared** [lllyasviel/Fooocus#3063] [Ideas]
Is possible to scale the prompt window as we can do with the Image render one? It becomes difficult to type if the text is longer than the prompt window.

**Comment by mashb1t** [comment]
I don't even know why this has been deliberately disabled. Adjusted, please check out https://github.com/lllyasviel/Fooocus/pull/3074

**Comment by PeterJared** [comment]
Thanks a lot!!

**Comment by PeterJared** [comment]
How can I get the branch? Perhaps there is a generic documentation for that as for now I only get automated updates

**Reply by mashb1t** [reply]
You have to add the remote for my fork, then pull the branch. But it will be merged to main tomorrow, so not really a need for you to do this.

**Comment by PeterJared** [comment]
great!
